internal/cmd/envs.go

```diff
@@ -51,6 +51,7 @@ func lsEnvsCommand() *cobra.Command {
 	var (
 		outputFmt string
 		user      string
+		provider  string
 	)
 
 	cmd := &cobra.Command{
```

Comment:
it would also be very nice if we included `Provider` as a table column in all calls to `envs ls`... I can follow this up with that change though.